source/css/scss-common/objects/_collapse.scss

```diff
@@ -496,27 +501,11 @@
 
 .a-list-parentRightHolder {
   &:not(.a-expanded) {
-    .a-list-expandedLink-href {
-      .expandable-icon {
-        .a-dropdownCircleArrow {
-          width: 24px;
-          min-width: 24px;
-          height: 24px;
-          margin-top: auto;
-          background: $blue-dark !important;
-          border: 3px solid $blue-dark !important;
-
-          i {
-            @include a-fontSize20;
-            margin-top: 0;
-          }
-        }
-      }
 
-      .hideText {
-        display: none;
-        }
-    }
+
```

Comment:
gir ikke dette en lint-feil siden det er to spaces? `npm run lint`

Reply:
fikset